README.md

```diff
@@ -111,6 +111,10 @@ gProfiler can be run in a continuous mode, profiling periodically, using the `--
 Note that when using `--continuous` with `--output-dir`, a new file will be created during *each* sampling interval.
 Aggregations are only available when uploading to the Granulate Performance Studio.
 
+### Uploading profiling data from another source
+gProfiler can be used to upload external .col file to Granulate Performance Studio by using `--upload-collapsed-file <path-to-.col>` flag.
+In case of gProfiler run volume mapping flag must be added. Example: `docker run --name granulate-gprofiler -v <path-to.col>:/home/outputdir/ --pid=host --userns=host --privileged  gprofiler:latest --token=<token> --service-name="<service>" --upload-collapsed-file /home/outputdir/<name>.col -u`
```

Comment:
UX wise, I think `--upload-collapsed-file` should be enough (so `-u` needn't be given if that one's given).

Alternatively - enforce having `-u` given if `--upload-collapsed-file` is given